MentorBook.Data/Models/Friend.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MentorBook.Data.Models
+{
+   public class Friend
+    {
+        public int Id { get; set; }
+        public string Email { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Phone { get; set; }
+        public DateTime DateOfBirth { get; set; }
+        public int HomeTownId { get; set; }
+        public string HomeTown { get; set; }
+        public int CurrentTownId { get; set; }
```

Comment:
In the database the town id's are nullable, so it is possible to have record with value null, which in this case of int will be converted to 0, which is wron.